src/Github/CreateReleaseTextThroughChangelog.php

```diff
@@ -109,14 +97,13 @@ private function normalizeChangelog(string $changelog, string $version): string
      */
     private function normalizeChangelogHeadings(string $changelog): string
     {
-        $lines         = explode("\n", trim($changelog));
+        $lines         = Str\split(Str\trim($changelog), "\n");
         $linesToRemove = [];
         foreach ($lines as $i => $line) {
-            $matches = [];
-
             // Does the line match one of the delimiter line types? If so,
             // capture that in $matches.
-            if (! preg_match('/^(?P<delim>-{3,}|={3,})$/', $line, $matches)) {
+            $matches = Regex\first_match($line, '/^(?P<delimiter>-{3,}|={3,})$/', Regex\capture_groups(['delimiter']));
```

Comment:
Confused by this: `$matches` is a `first_match()` here, but below we do `$matches['delimiter'][0];`?

Reply:
we are accessing an offset of a string, not array 🤷‍♂️ can use slice there if it makes things more clear, but it was always that way.

Reply:
That's alright, was just confused by the code, but the explanation suffices :+1: